example-apps/github/index.js

```diff
@@ -3,13 +3,6 @@ const issueCreate = require('./creates/issue');
 const issueTrigger = require('./triggers/issue');
 const authentication = require('./authentication');
 
-const handleHTTPError = (response, z) => {
-  if (response.status >= 400) {
```

Comment:
Assuming we added this to not throw on 3xx and GitHub doesn't throw >= 600, we no longer need this.